Project.toml

```diff
@@ -19,7 +19,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 CEnum = "0.4"
 EnzymeCore = "0.3"
 Enzyme_jll = "0.0.69"
-GPUCompiler = "0.19, 0.21"
+GPUCompiler = "0.19, 0.20, 0.21"
```

Comment:
I think we would need to do `0.21` the API might have changed.

Reply:
well lets at least see what CI yells at us about.